apps/aechannel/test/aesc_txs_SUITE.erl

```diff
@@ -150,17 +150,27 @@
          fp_missing_contract_address/1,
 
          fp_insufficent_tokens/1,
-         fp_too_soon/1
+         fp_too_soon/1,
+
+         % off-chain name registration not allowed
+         fp_register_name/1
         ]).
 
 -include_lib("common_test/include/ct.hrl").
 -include_lib("apps/aecore/include/blocks.hrl").
 -include_lib("stdlib/include/assert.hrl").
+-include_lib("apps/aecontract/src/aecontract.hrl").
 
 -define(MINER_PUBKEY, <<12345:?MINER_PUB_BYTES/unit:8>>).
 -define(BOGUS_CHANNEL, <<0:?MINER_PUB_BYTES/unit:8>>).
 -define(ROLES, [initiator, responder]).
--define(VM_VERSION, 1).
+-define(VM_VERSION, ?AEVM_01_Sophia_01).
+-define(TEST_LOG(Format, Data),
+        try ct:log(Format, Data)
```

Comment:
This seems used in the test only so the macro and the try-catch seem not needed, and ct:log can be called directly?